nutype_macros/src/lib.rs

```diff
@@ -40,135 +34,55 @@ fn expand_nutype(
         derive_traits,
     } = parse_meta(type_definition)?;
     match inner_type {
-        InnerType::String => {
```

Comment:
Could be refactored into something like this futher:

```rs
let meta = parse_meta(type_definition)?;
let (inner_type, meta2) = meta.to_meta2(attrs);

...

StringNewtype::expand(meta2)
```

However, one need to pick some good proper names instead of `meta2`.